src/components/SelectionListRadio/index.js

```diff
@@ -0,0 +1,46 @@
+import React, {forwardRef, useEffect, useState} from 'react';
+import {Keyboard} from 'react-native';
+import BaseSelectionListRadio from './BaseSelectionListRadio';
+import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
+
+const SelectionListRadio = forwardRef((props, ref) => {
+    const [isScreenTouched, setIsScreenTouched] = useState(false);
```

Comment:
Small performance improvement, this could be rewritten to `useRef`, as `isScreenTouched` is not used on UI so we don't need to re-render.

Reply:
@ArekChr Cool, I agree! It's like this because it was just copied from the old implementation. ccing @lukemorawski as he will be continuing with the refactor